src/test/scala/scalismo/utils/MemoizationTests.scala

```diff
@@ -28,14 +26,14 @@ class MemoizationTests extends ScalismoTestSuite {
 
     it("yields the same results after memoizing") {
       val testFunMemoized = Memoize(testFun, 100000)
-      for (x <- -2 * Math.PI until 2 * Math.PI by 0.1)
-        testFun(x) should be(testFunMemoized(x))
+      for (x <- BigDecimal(-2.0 * Math.PI) until (2.0 * Math.PI) by 0.1)
```

Comment:
Do we really need BigDecimal or could we cast the constant Math.PI to double instead? For me, this would be the more natural way.

Reply:
This is a deprecation from the scala side. If we want a step-size, which is fractional, we need a BigDecimal (at least this is what the compiler tells you)